src/dynamicResources/model/supported_resources.json

```diff
@@ -692,7 +704,7 @@
         "documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-eks-addon.html"
     },
     "AWS::EKS::Cluster": {
-        "operations": ["CREATE", "READ", "UPDATE", "DELETE", "LIST"],
+        "operations": ["READ", "CREATE", "UPDATE", "LIST", "DELETE"],
```

Comment:
why is the order different for this one?

Reply:
great question